src/OnboardingSPA/pages/Steps/BasicInfo/basicInfoForm.js

```diff
@@ -24,7 +24,7 @@ const BasicInfoForm = () => {
 
 	const [ siteTitle, setSiteTitle ] = useState( '' );
 	const [ siteDesc, setSiteDesc ] = useState( '' );
-	const [ siteLogo, setSiteLogo ] = useState( 0 );
+	const [ siteLogo, setSiteLogo ] = useState( { id: 0, url: '' } );
```

Comment:
If we are setting this data pattern in flows, why are we again setting it here? should we not be using the one from the backend? else if we decide to make `id 1` then we need to change the backend and frontend?